src/rp2_common/hardware_xosc/xosc.c

```diff
@@ -13,14 +13,21 @@
 #include "hardware/regs/xosc.h"
 #include "hardware/structs/xosc.h"
 
+// Allow lengthening startup delay to accommodate slow-starting oscillators
```

Comment:
Can you change this to a `// PICO_CONFIG: ` so that it shows up in the docs (see other places)

Reply:
Sure. Should I also do some kind of checking on the value to make sure the resultant multiplication doesn't overflow the register field of 14 bits? The `struct` definition for the register does not take that into account.

Reply:
![image](https://user-images.githubusercontent.com/2847802/120013197-77335680-bfae-11eb-96ea-b6b15ac58416.png)

https://github.com/raspberrypi/pico-sdk/blob/afc10f3599c27147a6f34781b7102d86f58aa5f6/src/rp2040/hardware_structs/include/hardware/structs/xosc.h#L15-L22

Reply:
yes makes sense - please follow invalid_params_if() style... (which will require you to add some macros to xosc.h - just copy/paste from somewhere else)